tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/createCveIssue.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/github"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/issueManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/retrier"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/version"
+)
+
+const (
+	backportFlag              = "backport"
+	announcementSourceUrlFlag = "announcementSourceUrl"
+	announcementBodyFlag      = "announcementBody"
+	issueAutocreatedTemplate  = "Issue was generated by eksGoTools createCveIssue, by:"
+)
+
+var (
+	announcementSourceUrl string
+	announcementBody      string
+
+	createCveIssue = &cobra.Command{
+		Use:   "createCveIssue [OPTIONS]",
+		Short: "Create new top level CVE Issue",
+		Long:  `Create a new top level CVE Issue in aws/eks-distro-build-tooling`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			owner := repositoryOwnerFlag
+			repo := repositoryFlag
+			//email := githubEmailFlag
+			//prBranch := commitBranch
+			//authorName := authorNameFlag
+
+			title := "Title"
+			label := []string{"security"}
+			assignee := "rcrozean"
+			state := "open"
+
+			retrier := retrier.New(time.Second*380, retrier.WithBackoffFactor(1.5), retrier.WithMaxRetries(15, time.Second*30))
+
+			token, err := github.GetGithubToken()
+			if err != nil {
+				return fmt.Errorf("getting Github PAT from environment at variable %s: %v", github.PersonalAccessTokenEnvVar, err)
+			}
+			githubClient, err := github.NewClient(cmd.Context(), token)
+			if err != nil {
+				return fmt.Errorf("setting up Github client: %v", err)
+			}
+
+			// set up PR Creator Handler
+			//o := &prmanager.Opts{
+			//	SourceOwner: owner,
+			//	SourceRepo:  repo,
+			//	PrRepo:      repo,
+			//	PrRepoOwner: owner,
+			//}
+			// prCreator := prmanager.New(retrier, githubClient, o)
+
+			// set up Issue Creator handler
+			issueManagerOpts := &issueManager.Opts{
```

Comment:
I think we can probably set these to consts -- e.g. `aws` and `eks-distro-build-tooling`, as that will be the only target for this cmd